source/includes/extracts-keyfile-info.yaml

```diff
@@ -24,22 +27,30 @@ source:
    ref: _keyfile-intro
 replacement:
    deployment: replica set
+   components: :program:`mongod` instances
 ---
 ref: keyfile-intro-sharded-cluster
 source:
    file: extracts-keyfile-info.yaml
    ref: _keyfile-intro
 replacement:
    deployment: sharded cluster
+   components: :program:`mongod` or :program:`mongos` instances
 ---
 ref: _keyfile-distribution
 content: |
-   Copy the keyfile to each server hosting the {{deployment}} members. Use a
-   consistent location for each server.
+
+   Copy the keyfile to each server hosting the {{deployment}} members. For
+   simplicity in configuration and maintenance, save the keyfile to a
+   consistent location for each server, such as the file path specified in
+   the {{programs}} :setting:`storage.dbPath` setting.
```

Comment:
Double checking that this recommendation of storage.dbPath has been vetted by someone.